frame/staking/src/tests.rs

```diff
@@ -5053,6 +5053,9 @@ fn min_commission_works() {
 }
 
 #[test]
+#[ignore]
+// TODO: This test should go away as we've decided that we don't cater for max_nominations decrease.
```

Comment:
it should not go away, the test is documenting what would happen if someone decreases `MaxNominations`. The test behavior, might be weird, but it should certainly still exist and make sense to us.

Reply:
This test does not succeed anymore, because we stopped catering for this scenario in Staking and removed the if Some check.

Reply:
I don't think we should have done it then. Lets revise this at the end again.